site/content/entry/yuzu-progress-report-apr-2020/index.md

```diff
@@ -0,0 +1,214 @@
++++
+date = "2020-05-15T20:28:00-03:00"
+title = "Progress Report April 2020"
+author = "GoldenX86"
+forum = 0
++++ 
+Hello awesome yuz-ers! 
+I hope you are doing well. 
+We have lots of stuff to cover in the April progress report. 
+I’ll cover some general new additions, bug and regression fixes, and groundwork for future stuff that affect both Mainline and 
+Early Access here, but first, some really big changes, some once again due to serendipity.
+<!--more-->
+
+## The Big Changes
+
+This month brought us huge improvements to compatibility, fidelity and resources management. To start with, 
+[bunnei](https://github.com/bunnei), our serendipitous master, fixed a softlock in `Final Fantasy VIII Remastered`. 
+What he didn’t know at the time is that this single small fix also made `Super Smash Bros. Ultimate` playable. Thanks to this you can enjoy local multiplayer battles with your friends, either locally or even via Parsec!
+
+For quite a while Nvidia Turing users (GTX 16 series and RTX 20 series GPUs) had to deal with several rendering bugs, like black terrain on `Fire Emblem: Three Houses`, or black glitches on `Super Monkey Ball: Banana Blitz HD`. 
```

Comment:
"Black glitches" is too general imho... I know it's complicated to describe that kind of weird stuff but maybe it's possible to find a better term